sonatype-ci-release/src/main/scala/org/typelevel/sbt/TypelevelSonatypeCiReleasePlugin.scala

```diff
@@ -57,9 +60,18 @@ object TypelevelSonatypeCiReleasePlugin extends AutoPlugin {
 
       tags ++ branches
     },
+    tlCreateJobSummary := {
+      Option(System.getenv("GITHUB_STEP_SUMMARY")).fold(
+        streams.value.log.error("GITHUB_STEP_SUMMARY is not defined")
+      ) { summaryFile =>
+        val buildVersion: String = (ThisBuild / version).value
+        IO.write(new File(summaryFile), buildVersion)
+      }
+    },
     githubWorkflowTargetTags += "v*",
     githubWorkflowPublish := Seq(
-      WorkflowStep.Sbt(List("tlRelease"), name = Some("Publish"))
+      WorkflowStep.Sbt(List("tlRelease"), name = Some("Publish")),
+      WorkflowStep.Sbt(List("tlCreateJobSummary"), name = Some("Create Job Summary"))
```

Comment:
```suggestion
      WorkflowStep.Sbt(List("tlRelease", "tlCreateJobSummary"), name = Some("Publish"))
```

Reply:
This config will result in `run: sbt '++ ${{ matrix.scala }}' tlRelease tlCreateJobSummary` that is not a valid syntax 🤔